model/metrics/nodejs-metrics.yaml

```diff
@@ -0,0 +1,98 @@
+groups:
+  - id: metric.nodejs.eventloop.delay.min
+    type: metric
+    metric_name: nodejs.eventloop.delay.min
+    brief: "Event loop minimum delay."
+    instrument: gauge
+    unit: "s"
+    stability: experimental
+    prefix: nodejs.eventloop.delay.min
+    attributes:
+      - ref: nodejs.version
+        requirement_level: recommended
+
+  - id: metric.nodejs.eventloop.delay.max
+    type: metric
+    metric_name: nodejs.eventloop.delay.max
+    brief: "Event loop maximum delay."
+    instrument: gauge
+    unit: "s"
+    stability: experimental
+    prefix: nodejs.eventloop.delay.max
+    attributes:
+      - ref: nodejs.version
+        requirement_level: recommended
+
+  - id: metric.nodejs.eventloop.delay.mean
+    type: metric
+    metric_name: nodejs.eventloop.delay.mean
+    brief: "Event loop mean delay."
+    instrument: gauge
+    unit: "s"
+    stability: experimental
+    prefix: nodejs.eventloop.delay.mean
+    attributes:
+      - ref: nodejs.version
+        requirement_level: recommended
+
+  - id: metric.nodejs.eventloop.delay.stddev
+    type: metric
+    metric_name: nodejs.eventloop.delay.stddev
+    brief: "Event loop standard deviation delay."
+    instrument: gauge
+    unit: "s"
+    stability: experimental
+    prefix: nodejs.eventloop.delay.stddev
+    attributes:
+      - ref: nodejs.version
+        requirement_level: recommended
+
+  - id: metric.nodejs.eventloop.delay.pfifty
```

Comment:
nit: any reason for the leafing `p` in `pfifty`? if so, why not then use it in the following percentiles (`pninety`, `pninety_nine`)?

Reply:
I added these as p50, but there is a limitation on the readme generation that couldn't read numbers, so I just switched to `fifty`, and then kept the `p`, since that is the name of the metric. I must have deleted on the others by accident, so I added back to keep consistent.
 
This value doesn't really matter, is just used on the readme generation, but doesn't even show on the readme itself when reading it.